src/docs/cookbook/navigation/navigation-basics.md

```diff
@@ -68,18 +71,15 @@ class SecondScreen extends StatelessWidget {
 
 ## 2. Navigate to the second screen using `Navigator.push`
 
-In order to Navigate to a new screen, we'll need to use the
-[`Navigator.push`](https://docs.flutter.io/flutter/widgets/Navigator/push.html)
-method. The `push` method will add a `Route` to the stack of routes managed by
-the Navigator!
+The [`Navigator.push`](https://docs.flutter.io/flutter/widgets/Navigator/push.html) method 
+will **add** a `Route` to the stack of routes managed by the Navigator.
 
 The `push` method requires a `Route`, but where does the `Route` come from?
 We can create our own, or use the [`MaterialPageRoute`](https://docs.flutter.io/flutter/material/MaterialPageRoute-class.html)
 out of the box. The `MaterialPageRoute` is handy because it transitions to the
 new screen using a platform-specific animation.
 
-In the `build` method of our `FirstScreen` Widget, we'll update the `onPressed`
-callback:
+In the `build` method of our `FirstScreen` Widget, update the `onPressed` callback:
```

Comment:
In the `build()` method of the `FirstScreen` widget, update the `onPressed()` callback: